script/release-components

```diff
@@ -119,7 +119,6 @@ main() {
 
   info "building flynn"
   git checkout --force --quiet $commit
-  git tag "v${version}"
```

Comment:
We tag via the GitHub API, so do we either need to pull the tag or create the local tag in `tag_release`?